### ISSUE TITLE
TypeError: (0 , m.createRoutes) is not a function

### ISSUE THREAD

**Issue by marcoseoane** [kuflash/react-router-sitemap#81]
Node Version: v8.12.0
Create-React-App

I have replicated the way it is used in the example, and get the error mentioned in the issue title, if there is any information that I have failed to provide that would be of use to you please notify me and I will take a look and list here.

routes.jsx 
`
import React from 'react';
import { Route, Redirect } from 'react-router';

export default (
	<Route>
		<Route path='/' />
		<Route path='/about' />
		<Route path='/projects'>
			<Route path=':projectName'>
				<Route path='view' />
			</Route>
		</Route>
		<Route path='/contacts' />
		<Route path='/auth' />
		<Redirect from='/home' to='/' />
		<Route path='*' />
	</Route>
);`

sitemap-builder.js

`require('babel-register');

const router = require('./routes').default;
const Sitemap = require('react-router-sitemap').default;

(
	new Sitemap(router)
		.build('http://recruitd.com')
		.save('./sitemap.xml')
);`

.babelrc 

`
{
	"ignore": [
		"node_modules/**/*"
	],
	"presets": [
		"es2015",
		"react"
	]
}`

![screen shot 2018-11-12 at 13 35 52](https://user-images.githubusercontent.com/28088807/48350719-f9253380-e67f-11e8-8fdc-34cdab601fea.png)

**Comment by kuflash** [comment]
@marcoseoane what version `react-router` do you use?

**Comment by marcoseoane** [comment]
> @marcoseoane what version `react-router` do you use?

 "react-router": "^4.2.0",

**Comment by kuflash** [comment]
@marcoseoane unfortunately at the moment 4 version is not supported. Now you can use this is [example](https://github.com/kuflash/react-router-sitemap/issues/71#issuecomment-437269350)

**Comment by marcoseoane** [comment]
> @marcoseoane unfortunately at the moment 4 version is not supported. Now you can use this is [example](https://github.com/kuflash/react-router-sitemap/issues/71#issuecomment-437269350)

Thanks for the response, I have tried again to copy the exact example given in the issue you linked me to. Now i get an error complaining about the use of the 'import' statement in the sitemap-builder.js file, I made sure to include all of the babel dependencies in the package.json + copied the .babelrc file in the repo example just to make sure and nothing I do seems to get rid of this error, any ideas as to where i'm going wrong here?

**Comment by kuflash** [comment]
@marcoseoane maybe try use expression `require('babel-register');`? Can you create repository with its error?

**Comment by marcoseoane** [comment]
> @marcoseoane maybe try use expression `require('babel-register');`? Can you create repository with its error?

yeah i did include this statement right at the top of sitemap-builder.js and no luck, could there be anything overriding babel possibly?

**Comment by Joseph7451797** [comment]
@marcoseoane I found that I forgot to address my `npm scripts` in the [example](https://github.com/kuflash/react-router-sitemap/issues/71#issuecomment-437269350).

Here is my `npm scripts`:
```json
{
  "scripts": {
    "build:sitemap": "cross-env NODE_ENV=production babel-node build-sitemap.js --presets=env,react,stage-2"
  }
}
```
When you are using `babel-node`, there is no need to require `babel-register` in your JS file.

**Comment by marcoseoane** [comment]
> @marcoseoane I found that I forgot to address my `npm scripts` in the [example](https://github.com/kuflash/react-router-sitemap/issues/71#issuecomment-437269350).
> 
> Here is my `npm scripts`:
> 
> ```json
> {
>   "scripts": {
>     "build:sitemap": "cross-env NODE_ENV=production babel-node build-sitemap.js --presets=env,react,stage-2"
>   }
> }
> ```
> 
> When you are using `babel-node`, there is no need to require `babel-register` in your JS file.

thanks for the response, while it prevented my previous errors from triggering i'm now getting this. 

![screen shot 2018-11-19 at 16 01 11](https://user-images.githubusercontent.com/28088807/48719089-8d5d4080-ec14-11e8-9797-8782caf4ec1c.png)

**Comment by marcoseoane** [comment]
> @marcoseoane I found that I forgot to address my `npm scripts` in the [example](https://github.com/kuflash/react-router-sitemap/issues/71#issuecomment-437269350).
> 
> Here is my `npm scripts`:
> 
> ```json
> {
>   "scripts": {
>     "build:sitemap": "cross-env NODE_ENV=production babel-node build-sitemap.js --presets=env,react,stage-2"
>   }
> }
> ```
> 
> When you are using `babel-node`, there is no need to require `babel-register` in your JS file.

thanks for the response, your script prevented the other errors I got from triggering but now I am getting this.

![screen shot 2018-11-19 at 16 01 11](https://user-images.githubusercontent.com/28088807/48719824-42442d00-ec16-11e8-93e0-3e16183b6ba3.png)

**Comment by francisrod01** [comment]
Related  #85.
